util/multisig/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "multisig"
+version = "0.1.0"
+license = "MIT"
+authors = ["Nervos Core Dev <dev@nervos.org>"]
+edition = "2018"
+
+[dependencies]
+secp256k1 = "0.12.2"
```

Comment:
I lose the context, why we wrap the secp256k1 in `util/crypto` and why we don't use the wrapper here?